### PR TITLE
[bug] Tabs controller can now update roles [ closes #191 ]

### LINKED ISSUE #191: [bug] tab server controller - updateTab is not updating roles
In the tab server controller, the updateTab function is not updating the roles.

### PR DESCRIPTION
**Pull request by marcoviappiani**
#191

### CODE REVIEW

**Comment by RRDuke** [comment]
LGTM

File: modules/core/server/controllers/tabs.controller.server.js [hunk 1]
```diff
@@ -131,7 +131,8 @@ function getTabById (req, res, next, id) {
   var tabQuery = {
     where: {
       id: id
-    }
+    },
+    include: [{model: db.Role}]
```
Comment (codydaig) [inline modules/core/server/controllers/tabs.controller.server.js]: Alphabetical order

File: modules/core/server/controllers/tabs.controller.server.js [hunk 2]
```diff
@@ -236,4 +238,25 @@ function updateTab (req, res) {
   function sendUpdatedTab (updatedTab) {
     res.send(updatedTab);
   }
+
+  function setRoles () {
+    if(req.body.visibleRoles) {
+
```
Comment (codydaig) [inline modules/core/server/controllers/tabs.controller.server.js]: Excess White Space

File: modules/core/server/controllers/tabs.controller.server.js [hunk 3]
```diff
@@ -236,4 +238,25 @@ function updateTab (req, res) {
...
+      if(req.body.visibleRoles.length === 0) {
+        req.body.visibleRoles.push(null);
+      }
+      return updateOneTab();
+    } else {
+      return;
+    }
+
+    function updateOneTab () {
+      return db.Tab.findOne({
```
Comment (codydaig) [inline modules/core/server/controllers/tabs.controller.server.js]: Nested Object

File: modules/core/server/controllers/tabs.controller.server.js [hunk 4]
```diff
@@ -236,4 +238,25 @@ function updateTab (req, res) {
...
+        where: {id: req.tab.id}
+      }).then(function (tab) {
```
Comment (codydaig) [inline modules/core/server/controllers/tabs.controller.server.js]: Nested Function

**Pull request merged**